job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy

```diff
@@ -55,6 +55,10 @@ pipeline{
         ${rhbaNightlyStage(CURRENT_PRODUCT_VERSION, CURRENT_PRODUCT_BRANCH, CURRENT_PRODUCT_CONFIG_BRANCH)}
         ${kogitoNightlyStage(KOGITO_CURRENT_PRODUCT_VERSION, KOGITO_CURRENT_PRODUCT_BRANCH, OPTAPLANNER_CURRENT_PRODUCT_VERSION, CURRENT_PRODUCT_VERSION, CURRENT_RHBA_VERSION_PREFIX, KOGITO_CURRENT_PRODUCT_CONFIG_BRANCH)}
 
+        // 7.67.x-blue
+        ${rhbaNightlyStage('8.0.0', '7.67.x-blue', NEXT_PRODUCT_CONFIG_BRANCH)}
+        ${kogitoNightlyStage('1.13.x-blue', '1.13.x-blue', OPTAPLANNER_NEXT_PRODUCT_VERSION, NEXT_PRODUCT_VERSION, NEXT_RHBA_VERSION_PREFIX, KOGITO_NEXT_PRODUCT_CONFIG_BRANCH)}
```

Comment:
is here also Optaplanner included in kogito pipeline? just curious

Reply:
it is @mareknovotny . see https://github.com/kiegroup/kogito-pipelines/blob/bb2753a190472962c682b2cd48fd3d98f9162ca3/.ci/jenkins/Jenkinsfile.prod.nightly#L65